change-notes/1.21/analysis-cpp.md

```diff
@@ -36,3 +36,4 @@
     - The taint tracking library now includes taint-specific edges for functions modeled in `semmle.code.cpp.models.interfaces.DataFlow`.
     - The taint tracking library adds flow through library functions that are modeled in `semmle.code.cpp.models.interfaces.Taint`. Queries can add subclasses of `TaintFunction` to specify additional flow.
 - There is a new `FoldExpr` class, representing C++17 fold expressions.
+- The predicate `TypeMention.toString()` has been simplified to always return the string "`type mention`".  This greatly improves performance when using `Expr.toString()` or its descendants.
```

Comment:
Typo: `Expr` should be `Element`.

Also, let's not overstate the performance implications of this change. I like to avoid mentioning performance improvements and regressions in the change notes because even though it varies between releases, performance stays within the same ballpark over time while the quality of our results goes up.

What's important for customers to know is that if they have qltests that query every `Element`, then those tests will need their expected output updated.